h/services/list_groups.py

```diff
@@ -81,34 +119,31 @@ def _open_groups(self, user=None, authority=None):
                       .all())
         return self._sort(groups)
 
-    def _private_groups(self, user=None):
-        """Return this user's private groups per user.groups."""
+    def _user_groups(self, user=None):
+        """Return all groups that this user is a member of regardless of type"""
 
         if user is None:
             return []
         return self._sort(user.groups)
 
-    def _parse_origin(self, uri):
-        """
-        Return the origin of a URI or None if empty or invalid.
+    def _private_groups(self, user=None):
```

Comment:
Note the difference here for `user.groups` (which used to be `private` only and can now contain `restricted` groups).